grizli/pipeline/auto_script.py

```diff
@@ -3,6 +3,7 @@
 """
 import os
 import inspect
+from this import d
```

Comment:
?

Reply:
No idea what this is. Removed here: 059feb865d4304f355e85be3990691c20593edc5